gensrc/thrift/Types.thrift

```diff
@@ -380,3 +380,7 @@ struct TUserIdentity {
     3: optional bool is_domain
 }
 
+const i32 TSNAPSHOT_REQ_VERSION1 = 3;
+const i32 TSNAPSHOT_REQ_VERSION2 = 4;
```

Comment:
add some log to describe the major version function.

Reply:
OK